docs/content/installation/install_compatibility_requirements.md

```diff
@@ -33,7 +33,7 @@ Complete these steps for each worker node in Kubernetes cluster to prepare your
 
```

Comment:
regarding step 2.i. - nvme is missing there.
also, is there a markdown that would allow linking to a numbered step (like a sub-section title)?

Reply:
@ArbelNathan  @roysahar  Please let me know if something is missing from 2.1 here. currently it is written: 
![image](https://user-images.githubusercontent.com/54570654/145711681-f9048173-c7a8-46a5-b46a-d1b0c1dc3c00.png)
we also have the following step:
![image](https://user-images.githubusercontent.com/54570654/145711704-efc045f5-d90c-4e66-bdd4-a6e04e183d4b.png)

Reply:
I think the `i.` is the step that should be highlighted, though nvme might be relevant to `ii.` as well

Reply:
@ArbelNathan @roysahar  should nvme be specified separately than Fibre Channel mentions in 2 i and ii above?

Reply:
will include #8 as part of #2. see updates

Reply:
updated